applications/GeoMechanicsApplication/custom_conditions/Pw_point_flux_condition.cpp

```diff
@@ -0,0 +1,53 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//
+//  Main authors:    Mohamed Nabi
+//                   John van Esch
+//
+
+#include "custom_conditions/Pw_point_flux_condition.hpp"
+
+namespace Kratos
+{
+
+template <unsigned int TDim, unsigned int TNumNodes>
+PwPointFluxCondition<TDim, TNumNodes>::PwPointFluxCondition() 
+    : PwCondition<TDim, TNumNodes>()
+{
+}
+
+template <unsigned int TDim, unsigned int TNumNodes>
+PwPointFluxCondition<TDim, TNumNodes>::PwPointFluxCondition(IndexType NewId, GeometryType::Pointer pGeometry)
+    : PwCondition<TDim, TNumNodes>(NewId, pGeometry)
+{
+}
+
+template <unsigned int TDim, unsigned int TNumNodes>
+PwPointFluxCondition<TDim, TNumNodes>::PwPointFluxCondition(IndexType               NewId,
+                                                            GeometryType::Pointer   pGeometry,
+                                                            PropertiesType::Pointer pProperties)
+    : PwCondition<TDim, TNumNodes>(NewId, pGeometry, pProperties)
+{
+}
+
+template <unsigned int TDim, unsigned int TNumNodes>
+PwPointFluxCondition<TDim, TNumNodes>::~PwPointFluxCondition() = default;
+
+template <unsigned int TDim, unsigned int TNumNodes>
+void PwPointFluxCondition<TDim, TNumNodes>::CalculateRHS(VectorType&        rRightHandSideVector,
+                                                         const ProcessInfo& CurrentProcessInfo)
```

Comment:
You may remove CurrentProcessInfo because it is not used in the function. Keep only const ProcessInfo&.

Reply:
@markelov208 Thank you for the review.
CurrentProcessInfo  is removed

Reply:
Correction: Removing CurrentProcessInfo gives errors under Linux, because it is override of the base class.
Brought it back in order to avoid the compiliation errors.